test/specs/gfm/gfm-spec.js

```diff
@@ -1,42 +1,27 @@
-var marked = require('../../../lib/marked.js');
+var tester = require('../marked-tester.js');
 var gfmSpec = require('./gfm.0.28.json')
-var HtmlDiffer = require('html-differ').HtmlDiffer,
-    htmlDiffer = new HtmlDiffer();
-var since = require('jasmine2-custom-message');
 
-var Messenger = function() {}
-
-Messenger.prototype.message = function(spec, expected, actual) {
-  return 'CommonMark (' + spec.section + '):\n' + spec.markdown + '\n------\n\nExpected:\n' + expected + '\n------\n\nMarked:\n' + actual;
-}
-
-Messenger.prototype.test = function(spec, section, ignore) {
-  if (spec.section === section && ignore.indexOf(spec.example) < 0) {
-    var shouldFail = ~ignore.indexOf(spec.example);
-    it('should ' + (shouldFail ? 'fail' : 'pass') + ' example ' + spec.example, function() {
-      var expected = spec.html;
-      var actual = marked(spec.markdown, { headerIds: false, xhtml: true });
-      since(messenger.message(spec, expected, actual)).expect(
-        htmlDiffer.isEqual(expected, actual)
-      ).toEqual(!shouldFail);
-    });
-  }
+var testOptions = {
+  gfm: true,
+  pedantic: false,
+  tables: true,
+  headerIds: false,
+  xhtml: true
 }
 
-var messenger = new Messenger();
-
 describe('GFM 0.28 Tables', function() {
   var section = 'Tables';
 
   // TODO: Verify exmaple 193 is valid and passing
+  // var shouldPassButFails = [];
```

Comment:
You can remove this comment

Reply:
...see last reply...Someone can uncomment that one, comment out the real one, and be set to go.